specification/components/schemas/ShortlistList.yaml

```diff
@@ -0,0 +1,50 @@
+title: A list of services representing the services available to the patient to have treatment at
+type: object
+required:
+  - meta
+  - resourceType
+  - entry
+properties:
+  meta:
+    type: object
+    required: 
+      - profile
+    properties:
+      profile:
+        type: array
+        items:
+          type: string
+          enum:
+            - 'https://fhir.nhs.uk/STU3/StructureDefinition/eRS-Shortlist-List-1'
+          example: 'https://fhir.nhs.uk/STU3/StructureDefinition/eRS-Shortlist-List-1'
+          default: 'https://fhir.nhs.uk/STU3/StructureDefinition/eRS-Shortlist-List-1'
+  contained:
+    type: array
+    items:
+      $ref: 'SearchCriteriaParameters.yaml'
+  resourceType:
+    type: string
+    enum:
+      - List
+    example: 'List'
+    default: 'List'
+  entry:
+    type: array
+    items:
```

Comment:
I think this attribute is named "item" rather than "items", at least according to (https://fhir.nhs.uk/STU3/StructureDefinition/eRS-Shortlist-List-1)

Reply:
this is fine, the entry schema contains property 'item' further down. 'items' here is the property of openapi object array. Meaning the possible items that can be in this array. In this example it is a single schema that has been inlined (as adviced by Adam earlier on)